.github/BOTMETA.yml

```diff
@@ -619,207 +684,306 @@ files:
-    - networking
-    - nxos
+      - networking
+      - nxos
   lib/ansible/plugins/action/onyx_config.py:
     maintainers: $team_onyx
     labels: networking
   lib/ansible/plugins/action/sros:
     maintainers: $team_networking
     labels: networking
   lib/ansible/plugins/action/vyos:
+    support: network
     maintainers: $team_networking
     labels: networking
-  lib/ansible/plugins/callback/unixy.py:
-    support: community
-    maintainers: akatch
+###############################
+# plugins/cache
+  lib/ansible/plugins/cache/__init__.py:
+    support: core
+  lib/ansible/plugins/cache/base.py:
+    support: core
+###############################
+# plugins/callbacks
+  lib/ansible/plugins/callback/__init__.py:
+    support: core
+  lib/ansible/plugins/callback/debug:
+    support: core
+  lib/ansible/plugins/callback/default:
+    support: core
   lib/ansible/plugins/callback/grafana_annotations.py:
-    support: community
     maintainers: rrey
+  lib/ansible/plugins/callback/junit:
+    support: core
+  lib/ansible/plugins/callback/minimal:
+    support: core
+  lib/ansible/plugins/callback/oneline:
+    support: core
+  lib/ansible/plugins/callback/profile:
+    support: core
   lib/ansible/plugins/callback/sumologic.py:
-    support: community
     maintainers: ryancurrah
+  lib/ansible/plugins/callback/tree:
+    support: core
+  lib/ansible/plugins/callback/unixy.py:
+    maintainers: akatch
+###############################
+# plugins/cliconf
+
+
   lib/ansible/plugins/cliconf/:
-    maintainers: $team_networking
     labels: networking
+  lib/ansible/plugins/cliconf/eos.py:
+    support: network
+    maintainers: $team_networking
   lib/ansible/plugins/cliconf/exos.py:
     maintainers: rdvencioneck
-    labels: networking
+  lib/ansible/plugins/cliconf/ios.py:
+    support: network
+    maintainers: $team_networking
+  lib/ansible/plugins/cliconf/iosxr.py:
+    support: network
+    maintainers: $team_networking
   lib/ansible/plugins/cliconf/ironware.py:
     maintainers: paulquack
-    labels: networking
+  lib/ansible/plugins/cliconf/junos.py:
+    support: network
+    maintainers: $team_networking
   lib/ansible/plugins/cliconf/nos.py:
     maintainers: $team_extreme
-    labels: networking
-  lib/ansible/plugins/cliconf/nxos.py:
+  lib/ansible/plugins/cliconf/nxos.ipy:
+    support: network
     maintainers: $team_nxos
-    labels:
-    - networking
+    labels: nxos
   lib/ansible/plugins/cliconf/onyx.py:
     maintainers: $team_onyx
-    labels: networking
   lib/ansible/plugins/cliconf/routeros.py:
     maintainers: heuels
-    labels: networking
   lib/ansible/plugins/cliconf/slxos.py:
     maintainers: $team_extreme
+  lib/ansible/plugins/cliconf/voss.py:
+    maintainers: $team_extreme
+    labels: networking
+  lib/ansible/plugins/cliconf/vyos.py:
+    support: network
+    maintainers: $team_networking
+###############################
+# plugins/connection
+  lib/ansible/plugins/connection/:
+    support: core
```

Comment:
Should not the following entries be added for `connection/oc.py`  ?
```
maintainers: chouseknecht maxamillion fabianvf flaper87 willthames # same as module_utils/k8s
support: community
```

Same for docker connection plugin (maintainers should be all cloud/docker namespace maintainers)

Reply:
Yup. Added
There will no doubt be extra files that are not classified correctly. I'll do some random spot checks on PRs once this has been merged.